server/src/main/java/org/opensearch/indices/recovery/RecoveryTarget.java

```diff
@@ -132,11 +127,6 @@ public CancellableThreads cancellableThreads() {
         return cancellableThreads;
     }
 
-    public Store store() {
```

Comment:
why are these removed? Are they now managed in the parent class?

Reply:
Yes, it's in ReplicationTarget